packages/@lwc/wds-core/src/browser/ssr/index.js

```diff
@@ -60,6 +60,17 @@ export async function render(componentUrl, componentProps) {
   return await task('render', url.href, componentProps);
 }
 
+export async function setHookSSR(componentUrl, updatedHooks) {
+  const url = new URL(componentUrl, document.location.origin);
+  const serializeHooks = (updatedHooks) => {
```

Comment:
Why is the function created, immediately invoked, and then thrown away? Why not just `const entries...` in the `setHookSSR` function body?

Reply:
Updated